be/src/http/action/stream_load.cpp

```diff
@@ -400,6 +400,10 @@ Status StreamLoadAction::_on_header(HttpRequest* http_req, StreamLoadContext* ct
         request.db = ctx->db;
         request.tbl = ctx->table;
         request.label = ctx->label;
+        // set timestamp
+        struct timeval tv;
```

Comment:
you can use function `UnixMicros` util/time.h
but the current implementation is not right now, could you fix it by the way?